acceptance/testdata/detector/container/custom_workspace/mock_app/run

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Where do we use this file?

Reply:
A very good question :)
I used it before making some changes.
Deleted.